LICENSE.txt

```diff
@@ -200,27 +200,3 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
-==========================================================================
```

Comment:
I have no idea why the JQuery license was attached to LICENSE.txt that I copied from Solr